database/src/config/index.ts

```diff
@@ -6,7 +6,7 @@ dotenv.config()
 const constants = {
   CONFIG_VERSION: {
     DEFAULT: 'DEFAULT',
-    EXPECTED: 'v1.2022-03-18',
+    EXPECTED: 'v7.2022-06-15',
```

Comment:
In database folder:
`git checkout master src/config/index.ts`
and commit

Reply:
done

Reply:
correct the database-version to the value of the master-branch (v1.2022-03-18)